src/styles/blocks/catalog.scss

```
@@ -17,9 +17,17 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
 
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    transition: transform $duration;
```

You've changed not only transform but color as well
```suggestion
    transition: transform $duration, color $duration;
```